pkg/upgrade/upgrade.go

```diff
@@ -262,25 +263,11 @@ func makeHelmResources(namespace, version, clusterName, repoURL string, values [
 	return []runtime.Object{helmRepository, helmRelease}, nil
 }
 
-func upgradeGitManifests(gitClient git.Git, cname, wegoEnterpriseManifests string, logger logger.Logger) error {
-	capiKeepPath := filepath.Join(git.WegoRoot, git.WegoAppDir, "capi", "templates", ".keep")
-	capiKeepContents := string(strconv.AppendQuote(nil, "# keep"))
-	kustomizationPath := git.GetSystemQualifiedPath(cname, models.SystemKustomizationPath)
-	wegoAppPath := git.GetSystemQualifiedPath(cname, models.WegoAppPath)
-	wegoEnterprisePath := git.GetSystemQualifiedPath(cname, WegoEnterpriseName)
+func upgradeGitManifests(gitClient git.Git, repoDir, clusterPath, cname, wegoEnterpriseManifests string, logger logger.Logger) error {
+	wegoEnterprisePath := filepath.Join(clusterPath, WegoEnterpriseName)
 
 	manifests := map[string]string{
```

Comment:
Am I right that this `manifests`//`for path, content` construction can be removed & replaced with a simple call to `gitClient.Write`?

Reply:
Yikes! Yes, there used to be multiple files written here and when I found I didn't need one of them, I removed it to test and forgot to to go back and clean this up. Thanks!